### PR TITLE
[MACAPA-2019] Adding Initial Event Settings

### PR DESCRIPTION
**Pull request by eujuniorweb**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

### CODE REVIEW

**Comment by mattstratton** [comment]
@MikeRosTX can you elaborate on what you need reviewed?

**Comment by mattstratton** [comment]
Also there is a file from the London event in this PR. Please remove it from this PR

**Comment by mattstratton** [comment]
Also why is the Chattanooga data file being changed?

File: content/events/2019-london/welcome.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Type = "welcome"
 aliases = ["/events/2019-london/"]
 Description = "devopsdays London 2019"
 +++
+
```
Comment (mattstratton) [inline content/events/2019-london/welcome.md]: This file should not be changed by another event. Please remove this file from the PR.

File: data/events/2019-chattanooga.yml [hunk 2]
```diff
@@ -18,8 +18,8 @@ enddate: 2019-11-12T23:59:59-05:00
 
 # Leave CFP dates blank if you don't know yet, or set all three at once.
 cfp_date_start: 2019-04-15T00:00:00-05:00 # start accepting talk proposals.
-cfp_date_end:  2019-08-12T00:00:00-05:00 # close your call for proposals.
-cfp_date_announce:  # inform proposers of status
+cfp_date_end: 2019-08-12T00:00:00-05:00 # close your call for proposals.
```
Comment (mattstratton) [inline data/events/2019-chattanooga.yml]: Why is the Chattanooga file being changed?

**Comment by mattstratton** [comment]
@eujuniorweb please see my comments on this PR. You have made changes to files for other events and will affect their pages. Please re-submit this PR with those changes removed.

File: content/events/2019-london/welcome.md (hunk 1 above)
Comment (eujuniorweb) [reply content/events/2019-london/welcome.md]: ok i will do that

File: data/events/2019-chattanooga.yml (hunk 2 above)
Comment (eujuniorweb) [reply data/events/2019-chattanooga.yml]: I was using as an example and I ended up changing it.

**Comment by MikeRosTX** [comment]
@eujuniorweb we understand that you might have plenty of experience at this, but if you have not read the [**CONTRIBUTING.md**](https://github.com/devopsdays/devopsdays-web/blob/master/CONTRIBUTING.md) then please review it. Because small changes like the ones you made could have a **HUGE** impact on other city's events, like 3 in this case.

**Pull request closed without merging**